resources/public/api/conf/P1/application.raml

```diff
@@ -24,6 +24,29 @@ uses:
   annotations: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/annotations.raml
   types: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/types.raml
 
+/individuals/income:
```

Comment:
Not sure if this `/individuals/income` is required. In `individuals-employments-api` we just have `/match/{matchId}` see [here](https://github.com/hmrc/individuals-employments-api/pull/2/files#diff-40ffa559c8155d43219d2da7ca88889dR27).

Reply:
I would have thought that this needed by WSO2. In `individuals-api` we have: 
```
/individuals
  /match
  /{matchId}
```

Reply:
You're probably right, in which case we need to change it in `individuals-employments-api`. Let's see if monsieur @tom59 can confirm?

Reply:
scratch that, I know what the difference is. In the documentation, the endpoint's URL is currently displayed as: `/individuals/income/match/{matchId}`.
In the case of `hello-world` we are not adding the context, and the endpoint's URL is displayed as: `/hello-world`.
I think we should include the context, otherwise our consumers will be shown incorrect URLs.

Reply:
Do you agree with the above, @tom59?

Reply:
yes actually it is required. Can you add it in your @edinhodzic ?
The context is there in the other API such as https://github.com/hmrc/individuals-api/blob/master/resources/public/api/conf/1.0/application.raml#L27